### PR TITLE
feat(skin): update design tokens

### PR DESCRIPTION
**Pull request by tuentisre**
Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action

### CODE REVIEW

File: src/skins/vivo-new.tsx [hunk 1]
```diff
@@ -24,6 +24,7 @@ export const palette = {
     orangeLight40: '#FFB84C',
     pink: '#EB3D7D',
     pepper: '#CC1F59',
+    pepper10: '#FFEBED',
```
Comment (marcoskolodny) [inline src/skins/vivo-new.tsx:27]: @yceballost had to add this for vivo and vivo-new. Is this correct?

File: src/skins/o2-classic.tsx [hunk 2]
```diff
@@ -114,6 +114,9 @@ export const getO2ClassicSkin: GetKnownSkin = () => {
             buttonDangerBackgroundHover: palette.pepper55,
             buttonLinkBackgroundSelected: palette.o2SkyBlueLight10,
             buttonLinkBackgroundInverseSelected: applyAlpha(palette.white, 0.2),
+            buttonLinkDangerBackgroundSelected: palette.pepper10,
```
Comment (marcoskolodny) [inline src/skins/o2-classic.tsx:117]: @yceballost in o2-classic the new tokens didn't exist. I copied the ones from o2. Verify if this makes sense

File: src/button.css.ts [hunk 3]
```diff
@@ -421,23 +421,23 @@ const dangerLinkInverse: ComplexStyleRule = [
     link,
     sprinkles({
         color: vars.colors.textLinkDanger,
```
Comment (marcoskolodny) [inline src/button.css.ts:423]: @yceballost this should be textLinkDangerInverse according to specs, but that token doesn't exist :(

File: src/skins/o2-classic.tsx (hunk 2 above)
Comment (yceballost) [reply src/skins/o2-classic.tsx:115]: we haven't support o2-classic in mistica-design for a long time ago 😬

File: src/button.css.ts (hunk 3 above)
Comment (yceballost) [reply src/button.css.ts:423]: I think textLinkDangerInverse is not needed 🤔

**Pull request merged**